modules/cassandra/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.testcontainers</groupId>
+        <artifactId>testcontainers-parent</artifactId>
+        <version>0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>cassandra</artifactId>
+    <name>TestContainers :: Cassandra</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>database-commons</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <!--Database driver-->
+        <dependency>
+            <groupId>com.datastax.cassandra</groupId>
+            <artifactId>cassandra-driver-core</artifactId>
+            <version>3.3.2</version>
+        </dependency>
+
+        <!--Netty dependencies for docker tests-->
```

Comment:
Datastax Cassandra driver has its own Netty dependency: https://github.com/datastax/java-driver/blob/57724bd63c3038e3728260c7640abcea06f33b86/driver-core/pom.xml#L38 
which overrides transitive Netty dependency from com.github.docker-java.

Netty from Datastax driver has 4.0.47 driver
Netty from com.github.docker-java (which is testcontainers core dependency) has 4.1.11.Final version. 
So it all leads to behavior, that in Cassandra tests Docker container cannot be started due to absence of io.netty.channel.ChannelFactory  (4.0.47 has this interface in different io.netty.bootstrap package).

To resolve this jar purgatory, I use explicit Netty dependency for Cassandra module tests with version same as in com.github.docker-java.
In production (import jar testcontainers core jar and new cassandra jar) there is no com.github.docker-java dependency, so there is no problem with different Netty versions, so it's only transitive Netty from Datastax, which works fine with everything. 


Datastax has Jira ticket to update Netty version, but it will only be in 4.x version, which is not available yet: https://datastax-oss.atlassian.net/browse/JAVA-1241